scripts/publish-npm.ts

```diff
@@ -25,108 +25,150 @@
 import * as argparse from 'argparse';
```

Comment:
Note to reviewer: Just review this as a new file. It shares very little logic with the previous implementation.